test/ci/backend/test_lava.py

```diff
@@ -300,7 +300,26 @@
     'status': 'Complete',
     'id': 1234,
     'definition': yaml.dump(JOB_DEFINITION),
-    'multinode_definition': ''
+    'multinode_definition': '',
+    'start_time': '2020-11-13T10:23:50.019407Z',
+    'end_time': '2020-11-13T10:23:41.265697Z'
```

Comment:
Ends before starting?

Reply:
who knows, maybe it travels faster than light? :)

Reply:
`job name: back to the future`